### PR TITLE
Update Model3DLayout.py

### PR DESCRIPTION
**Pull request by PipKat**
doc/edits_to_Model3DLayout_py_in_modeler

### CODE REVIEW

File: pyaedt/modeler/Model3DLayout.py [hunk 1]
```diff
@@ -144,23 +148,23 @@ def colinear_heal(self, selection, tolerance=0.1):
...
-            type of expansion: ``ROUND``, ``MITER``, ``CORNER``
-        replace_original
+            Name of the object to expand.
+        size: float, optional
+            Size of the expansion. The default is ``1``.
+        expand_type: str, optional
+            Type of the expansion. Options are ``"ROUND"``, ``"MITER"``, and
+            ``"CORNER"``. The default is ``"ROUND"``.
+        replace_original : bool, optional
+             Whether to replace the orginal object. The default is ``False``, in which case
```
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:161]:
```suggestion
             Whether to replace the original object. The default is ``False``, in which case 
```

File: pyaedt/modeler/Model3DLayout.py [hunk 2]
```diff
@@ -361,18 +377,22 @@ def intersect(self, objectlists):
...
+        
         Parameters
         ----------
         objectlists : list
+            Listof elements to duplicate.
         count : int
+        
         direction_vector : list
-            x, y direction vector
+            List of `[x, y]` coordiantes for the direction vector.
```
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:389]:
```suggestion
            List of `[x, y]` coordinates for the direction vector.
```

File: pyaedt/modeler/Model3DLayout.py [hunk 3]
```diff
@@ -37,7 +43,14 @@ def __init__(self, parent):
 
     @property
     def edb(self):
-        """ """
+        """EBD.
+        
+        Returns
+        -------
+        :class:'pyaedt.Edb`
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:50]: Nice work.  Exactly like this.

File: pyaedt/modeler/Model3DLayout.py [hunk 4]
```diff
@@ -272,20 +285,21 @@ def import_ipc2581(self, ipc_filename, edb_path=None, edb_name=None):
 
     @aedt_exception_handler
     def subtract(self, blank, tool):
-        """Subtract objects from names
+        """Subtract objects from one or more names.
 
         Parameters
         ----------
         blank :
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:279]: str type

File: pyaedt/modeler/Model3DLayout.py [hunk 5]
```diff
@@ -333,18 +348,19 @@ def unite(self, objectlists):
...
     def intersect(self, objectlists):
-        """Intersect objects from names
+        """Intersect objects from names.
 
         Parameters
         ----------
-        objectlists :
-            list of objects to unite
+        objectlists : list
+            list of objects to intersect.
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:356]: Capitalize list.

File: pyaedt/modeler/Model3DLayout.py [hunk 6]
```diff
@@ -361,18 +377,22 @@ def intersect(self, objectlists):
...
     @aedt_exception_handler
     def duplicate(self, objectlists, count, direction_vector):
-        """
-        Duplicate one or multiple element along vector
+        """Duplicate one or more elements along a vector.
+        
         Parameters
         ----------
         objectlists : list
+            Listof elements to duplicate.
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:385]: Split list and of.

File: pyaedt/modeler/Model3DLayout.py [hunk 7]
```diff
@@ -52,37 +65,26 @@ def edb(self):
...
     @aedt_exception_handler
     def fit_all(self):
-        """ """
+        """ Fit all."""
         self.oeditor.ZoomToFit()
 
     @property
     def model_units(self):
-        """ """
+        """Model units. """
```
Comment (akaszynski) [inline pyaedt/modeler/Model3DLayout.py:83]: Remove unnecessary white space.

**Review by akaszynski — APPROVED** [review]
Approved with comments.  At the store and unable to make suggestions on mobile app.

File: pyaedt/modeler/Model3DLayout.py [hunk 8]
```diff
@@ -52,37 +65,26 @@ def edb(self):
...
     @property
     def oeditor(self):
-        """ """
+        """Layout."""
         return self._parent.odesign.SetActiveEditor("Layout")
 
     @aedt_exception_handler
     def fit_all(self):
-        """ """
+        """ Fit all."""
```
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:78]:
```suggestion
        """Fit all."""
```

File: pyaedt/modeler/Model3DLayout.py (hunk 7 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:83]:
```suggestion
        """Model units."""
```

File: pyaedt/modeler/Model3DLayout.py (hunk 4 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:292]:
```suggestion
        blank : str
```

File: pyaedt/modeler/Model3DLayout.py (hunk 5 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:356]:
```suggestion
            List of objects to intersect.
```

File: pyaedt/modeler/Model3DLayout.py (hunk 6 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py:385]:
```suggestion
            List of elements to duplicate.
```

File: pyaedt/modeler/Model3DLayout.py (hunk 5 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py]:
```suggestion
            List of objects to intersect.
```

File: pyaedt/modeler/Model3DLayout.py (hunk 6 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py]:
```suggestion
            List of elements to duplicate.
```

File: pyaedt/modeler/Model3DLayout.py (hunk 5 above)
Comment (PipKat) [inline pyaedt/modeler/Model3DLayout.py]:
```suggestion
            List of objects to intersect.
```

**Pull request merged**